core/eval.go

```diff
@@ -863,95 +915,117 @@ func evalQWATCH(args []string, c *Client) []byte {
 	return RESP_OK
 }
 
+// evalCOMMAND evaluates COMMAND <subcommand> command based on subcommand
+// COUNT: return total count of commands in Dice.
+func evalCOMMAND(args []string) []byte {
+	if len(args) == 0 {
+		return Encode(fmt.Errorf("(error) ERR wrong number of arguments for 'command' command"), false)
```

Comment:
nit: we could use errors.New() instead of fmt.Errorf().

There's not much difference, but since there are no custom args for this string we probably don't need fmt.Errorf